### PR TITLE
Update schedule for `GBC 2022 Spring` Semifinal.

### PR DESCRIPTION
**Pull request by RushFTK**
Also: 
- update results for quaterfinal.
- update last mappools
- update ruleset fix (SF & GF using same pools and will become BO13)

## Self-check

- [x] The changes are tested against the [contribution checklist](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#self-check)

### CODE REVIEW

File: wiki/Tournaments/GBC/GBC_2022_Spring/en.md [hunk 1]
```diff
@@ -375,10 +414,12 @@ Sunday, April 10, 2022:
...
 
-1. There will be one mappool for each stage.
-   - Regular stage Round 1 and Round 2 are seen as a single stage, meaning they use the same mappool.
+1. There will be three seprate mappools for all stage.
+   - Regular stage Round 1 and Round 2 use the mappool #1.
+   - Regular stage Round 3 and Quarterfinal use the mappool #2.
+   - Semifinal and Final use the mappool #3
 2. Each mappool consists of a fixed amount of maps each stage which will all be played under FreeMod conditions. This means that there is a unique FreeMod bracket.
-3. All mappools contain 13 beatmaps.
+3. mappool #1 & #2 contain 13 beatmaps, mappool #3 contain 15 beatmaps.
```
Comment (Walavouchey) [inline wiki/Tournaments/GBC/GBC_2022_Spring/en.md:422]:
```suggestion
3. Mappools #1 & #2 contain 13 beatmaps, and mappool #3 contains 15 beatmaps.
```

File: wiki/Tournaments/GBC/GBC_2022_Spring/en.md [hunk 2]
```diff
@@ -375,10 +414,12 @@ Sunday, April 10, 2022:
 
 ### Mappool instructions
 
-1. There will be one mappool for each stage.
-   - Regular stage Round 1 and Round 2 are seen as a single stage, meaning they use the same mappool.
+1. There will be three seprate mappools for all stage.
```
Comment (Walavouchey) [inline wiki/Tournaments/GBC/GBC_2022_Spring/en.md:417]:
```suggestion
1. There will be three separate mappools.
```

File: wiki/Tournaments/GBC/GBC_2022_Spring/en.md [hunk 3]
```diff
@@ -375,10 +414,12 @@ Sunday, April 10, 2022:
...
+   - Regular stage Round 1 and Round 2 use the mappool #1.
```
Comment (Walavouchey) [inline wiki/Tournaments/GBC/GBC_2022_Spring/en.md:418]:
```suggestion
   - Regular stage Round 1 and Round 2 use mappool #1.
```

File: wiki/Tournaments/GBC/GBC_2022_Spring/en.md [hunk 4]
```diff
@@ -375,10 +414,12 @@ Sunday, April 10, 2022:
...
+   - Regular stage Round 3 and Quarterfinal use the mappool #2.
```
Comment (Walavouchey) [inline wiki/Tournaments/GBC/GBC_2022_Spring/en.md:419]:
```suggestion
   - Regular stage Round 3 and Quarterfinals use mappool #2.
```

File: wiki/Tournaments/GBC/GBC_2022_Spring/en.md [hunk 5]
```diff
@@ -375,10 +414,12 @@ Sunday, April 10, 2022:
...
+   - Semifinal and Final use the mappool #3
```
Comment (Walavouchey) [inline wiki/Tournaments/GBC/GBC_2022_Spring/en.md:420]:
```suggestion
   - Semifinals and Finals use mappool #3.
```

File: wiki/Tournaments/GBC/GBC_2022_Spring/en.md [hunk 6]
```diff
@@ -342,11 +381,11 @@ Sunday, April 10, 2022:
 ### Playoff stage instructions
 
 1. Following the Regular stage, a single elimination stage will be played. This means that the winner continues with their runs, and the losing team will be eliminated from the tournament, except in the bronze match.
-2. Teams must play 2 matches on week 2 of the playoffs.
 
 ### Win conditions
 
-- In all stages, teams need to win 6 maps to win a match (best of 11).
+- In Regular stages and Quarterfinal, teams need to win 6 maps to win a match (best of 11).
```
Comment (Walavouchey) [inline wiki/Tournaments/GBC/GBC_2022_Spring/en.md:387]:
```suggestion
- In the Regular stage and Quarterfinals, teams need to win 6 maps to win a match (best of 11).
```

File: wiki/Tournaments/GBC/GBC_2022_Spring/en.md [hunk 7]
```diff
@@ -342,11 +381,11 @@ Sunday, April 10, 2022:
...
+- In Semifinal and Final , teams need to win 7 maps to win a match (best of 13).
```
Comment (Walavouchey) [inline wiki/Tournaments/GBC/GBC_2022_Spring/en.md:388]:
```suggestion
- In Semifinals and Finals, teams need to win 7 maps to win a match (best of 13).
```

File: wiki/Tournaments/GBC/GBC_2022_Spring/en.md [hunk 8]
```diff
@@ -87,25 +87,48 @@ The GB Cup 2022 Spring is run by the GB Team and various community members.
...
-### Regular stage week 3
+### Semifinal & Final
+
+**[Download the mappack here (130.7 MB)](https://drive.google.com/file/d/1GdRG5ay0LU_RaAfcpml8qG_Gu3bdlWB1/view)**
+
+- Rice
+  1. [DJ TOTTO - Chaplet (riunosk) \[Deceptive\]](https://osu.ppy.sh/beatmapsets/1440718#mania/2964130)
+  2. [Hachioji-P feat. Hatsune Miku - Blue Star (\[Crz\]Rachel) \[Sparkle Star\]](https://osu.ppy.sh/beatmapsets/1000382#mania/2093764)
+  3. [goreshit - satori de pon! (SaltedFISSH) \[\~\~Pon!\~\~\]](https://osu.ppy.sh/beatmapsets/1251545#mania/2601285)
+  4. [DECO*27 - Android Girl feat. Hatsune Miku (Xingyue -) \[AI x1.1\]](https://osu.ppy.sh/beatmapsets/969824#mania/2029268)
```
Comment (Walavouchey) [inline wiki/Tournaments/GBC/GBC_2022_Spring/en.md:114]:
```suggestion
  4. [DECO\*27 - Android Girl feat. Hatsune Miku (Xingyue -) \[AI x1.1\]](https://osu.ppy.sh/beatmapsets/969824#mania/2029268)
```

File: wiki/Tournaments/GBC/GBC_2022_Spring/en.md [hunk 9]
```diff
@@ -87,25 +87,48 @@ The GB Cup 2022 Spring is run by the GB Team and various community members.
 | 15 | **GBC最强阵容9个Miyes** | ![][flag_CN] **[miyes](https://osu.ppy.sh/users/12942073)**, ![][flag_CN] [alivn](https://osu.ppy.sh/users/15807665), ![][flag_CN] [ZOM148](https://osu.ppy.sh/users/10402690), ![][flag_CN] [ChiralSS](https://osu.ppy.sh/users/18211706), ![][flag_CN] [1atk](https://osu.ppy.sh/users/20520767) |
 | 16 | **头号倒霉蛋** | ![][flag_CN] **[StarTemplar](https://osu.ppy.sh/users/8795096)**, ![][flag_CN] [9961](https://osu.ppy.sh/users/14609209), ![][flag_CN] [1145810](https://osu.ppy.sh/users/24449770), ![][flag_CN] [lochid](https://osu.ppy.sh/users/18107639), ![][flag_CN] [OTruthO](https://osu.ppy.sh/users/17470634) |
 
-## Match schedule: Playoffs week 1
+## Match schedule: Semifinal
```
Comment (Walavouchey) [inline wiki/Tournaments/GBC/GBC_2022_Spring/en.md:90]:
```suggestion
## Match schedule: Semifinals
```

File: wiki/Tournaments/GBC/GBC_2022_Spring/en.md [hunk 10]
```diff
@@ -87,25 +87,48 @@ The GB Cup 2022 Spring is run by the GB Team and various community members.
...
 
 | Team A | Team B | Match time |
 | --: | :-- | :-: |
-| 货拉拉拉不拉拉布拉多 | 涩涩不队 | May 2 (Mon) 19:00 (UTC+8) |
+| GBC最强阵容9个Miyes | 金戈铁马的鲜丰水果摊 | May 8 (Sun) 18:00 (UTC+8) |
 
 ## Mappools
 
-### Regular stage week 3
+### Semifinal & Final
```
Comment (Walavouchey) [inline wiki/Tournaments/GBC/GBC_2022_Spring/en.md:106]:
```suggestion
### Semifinals & Finals
```

**Review by Walavouchey — COMMENTED** [review]
i guess playoffs week 2 has been renamed? the tournament schedule at the top needs to be edited too

File: wiki/Tournaments/GBC/GBC_2022_Spring/en.md [hunk 11]
```diff
@@ -153,6 +176,22 @@ The GB Cup 2022 Spring is run by the GB Team and various community members.
 
 ## Match results
 
+### Quarterfinal
```
Comment (RushFTK) [inline wiki/Tournaments/GBC/GBC_2022_Spring/en.md:179]:
```suggestion
### Quarterfinals
```

**Comment by RushFTK** [comment]
> i guess playoffs week 2 has been renamed? the tournament schedule at the top needs to be edited too

@Walavouchey well, the tournament was set to play SF and Finals on same weekend but since this Saturday is weekday in china so finals are *rescheduled* to next week. I'm not sure should tournament schedule at the top be updated in such situation.

**Comment by Walavouchey** [comment]
yeah it should be updated if it otherwise wouldn't reflect the actual schedule

File: wiki/Tournaments/GBC/GBC_2022_Spring/en.md [hunk 12]
```diff
@@ -87,25 +88,48 @@ The GB Cup 2022 Spring is run by the GB Team and various community members.
...
+- SV
+  1. [Cutter - EmoCosine (\[GB\]AelSan) \[Miscedence's Cute \[GBC edit\]\]](https://osu.ppy.sh/beatmapsets/1755418#mania/3592340)
+  2. [TSAR - 0.0? (\[GB\]AelSan) \[Mafufu & AelSan's @v@!\]](https://osu.ppy.sh/beatmapsets/1755419#mania/3592342)
+- Hybrid
+  1. [IOSYS - Ringo's Tea Party (TheToaphster) \[Spooked\]](https://osu.ppy.sh/beatmapsets/638838#mania/1355027)
+  2. [Camellia feat. Nanahira - Ska-sh All Neurons!! (HowToPlayLN) \[Re-forward !!\]](https://osu.ppy.sh/beatmapsets/1122764#mania/2349915)
+- Tiebreaker
+  1. **[Camellia - SECRET BOSS (\[GB\]yobrevelc) \[Glorious Battlefield of Champions\]](https://osu.ppy.sh/beatmapsets/1755452#mania/3592441)**
+
+### Regular stage week 3 & Quarterfinal
```
Comment (Walavouchey) [inline wiki/Tournaments/GBC/GBC_2022_Spring/en.md:132]:
```suggestion
### Regular stage week 3 & Quarterfinals
```

**Review by Walavouchey — APPROVED** [review]

**Pull request merged**